interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -23,6 +24,11 @@ function __construct($template_mod = "general") {
       $this->assign("gbl_vitals_options",$GLOBALS['gbl_vitals_options']);
     }
 
+		function setFormId($form_id){
```

Comment:
Apologies, I was looking at the wrong function and yes the setFormId($form_id) was added on this commit.  It was added because I did not know what this function did $c->setFormId($form_id); which is in the forms\vitals\new.php file. And I was told that this type of code was old. It was further suggested to add the setFormId function in the C_FormVitals.class.php file. It was hard to figure out where the variable was being filled. So once I figured out it was zero then I was able to finish the substitution and remove the variable from being passed in the method so that the method matched the parent.